bakery/env/staging.json

```diff
@@ -15,5 +15,5 @@
   "GOOGLE_FOLDER_ID": "",
   "GDOC_QUEUE_FILENAME": "",
   "GDOC_FEED_FILE_URL": "",
-  "S3_DIST_PREVIEW_BUCKET": "ce-cops-dist-preview-staging"
+  "S3_DIST_PREVIEW_BUCKET": "openstax-sandbox-cops-artifacts"
```

Comment:
Ah! I forgot! `S3_PDF_BUCKET` also should point to this, right?

More specifically, there should only be one option (`S3_COPS_ARTIFACTS_BUCKET`, maybe?) and it should cover both PDF and DIST_PREVIEW, since they should always be the same.